worker/src/main/scala/org/apache/celeborn/service/deploy/worker/PushDataHandler.scala

```diff
@@ -755,6 +756,90 @@ class PushDataHandler extends BaseMessageHandler with Logging {
     }
   }
 
+  def replicateMapPartition(
+      shuffleKey: String,
+      location: PartitionLocation,
+      messageType: Type,
+      message: Message,
+      wrappedCallback: WrappedRpcResponseCallback): Unit = {
+    _worker.getMapPartitionReplicatePool(
```

Comment:
For ReducePartition, replicate pool is defined in PushHandler, why not keep consistent for MapPartition?

Reply:
it defines a function val MapPartitionReplicatePool to keep consistent for reducePartition